v2.1/build-a-ruby-app-with-cockroachdb-activerecord.md

```diff
@@ -33,22 +33,20 @@ To install ActiveRecord as well as the [pg driver](https://rubygems.org/gems/pg)
 $ gem install activerecord pg activerecord-cockroachdb-adapter
 ~~~
 
-{{site.data.alerts.callout_success}}
-The exact command above will vary depending on the desired version of ActiveRecord. Specifically, version 4.2.x of ActiveRecord requires version 0.1.x of the adapter; version 5.1.x of ActiveRecord requires version 0.2.x of the adapter.
-{{site.data.alerts.end}}
+{{site.data.alerts.callout_success}}The exact command above will vary depending on the desired version of ActiveRecord. Specifically, version 4.2.x of ActiveRecord requires version 0.1.x of the adapter; version 5.1.x of ActiveRecord requires version 0.2.x of the adapter.{{site.data.alerts.end}}
```

Comment:
How do I get the 0.2.x version of the adapter? 
(Here's the error I get: 
in `raise_if_conflicts': Unable to activate activerecord-cockroachdb-adapter-0.2.3, because pg-1.0.0 conflicts with pg (< 0.22, >= 0.20) (Gem::ConflictError))

Reply:
I got this error too (but I did not get it several weeks ago, huh).  Strangely, when you do `gem install pg` it installs both 0.2 and 1.0 -- at least on my machine.

Was able to fix this and get it working by changing the order of the imports in the file `activerecord-basic-sample.rb` to list the `pg` module last.  🤷‍♂️  Yay computers!